test/general/test_implemented.py

```diff
@@ -47,3 +48,8 @@ def test_slot_data(self):
                 for key, data in multiworld.worlds[1].fill_slot_data().items():
                     self.assertIsInstance(key, str, "keys in slot data must be a string")
                     self.assertIsInstance(encode(data), str, f"object {type(data).__name__} not serializable.")
+
+    def test_no_failed_world_loads(self):
+        if failed_world_loads:
+            self.fail(f"The following worlds failed to load: {failed_world_loads}")
+            
```

Comment:
extra newline at end of file

Reply:
There's only supposed to be one newline at the end of the file right? There's only one. I think initially pushing without it seems to confuse Github for some reason.

Reply:
Nevermind, Github Desktop is what got confused.